change/@ni-nimble-components-573fc816-eaa7-4e07-b0ef-8bfd5ccff19e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
This change ended up improving our aria score from 81 to 84 in lighthouse @m-akinc @jattasNI 🎉

Looks like in numeric increment / decrement buttons we were setting tabindex="-1" but that wasn't reflect to the inner button which was then technically focusable. After this change it also gets tabindex="-1" and makes lighthouse happy ✔

![image](https://github.com/ni/nimble/assets/1588923/ac25dc35-fda0-4b34-8221-19b0f25ec02b)